server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QGatesWs.java

```diff
@@ -79,8 +79,8 @@ static void addConditionParams(NewAction action) {
       .setPossibleValues(QualityGateConditionDto.ALL_OPERATORS);
 
     action.createParam(PARAM_PERIOD)
-      .setDescription("Condition period")
-      .setExampleValue("1");
+      .setDescription("Condition period. If not set, the absolute value is considered.")
```

Comment:
for the time being, I won't change anything because:
- it doesn't suit the workflow where api/qualitygates/app is called to have the available period with labels etc.
- it can be changed later